scripts/report_status.sh

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
```

Comment:
Is it not used at all anywhere ?

Reply:
no, we no longer set status from lava, instead we set status from github workflow - much cleaner